netbox/virtualization/views.py

```diff
@@ -350,7 +351,8 @@ class VirtualMachineListView(generic.ObjectListView):
 
 @register_model_view(VirtualMachine)
 class VirtualMachineView(generic.ObjectView):
-    queryset = VirtualMachine.objects.all()
+    queryset = VirtualMachine.objects.all().annotate(
+        disk_size=Sum('virtualdisks__size'))
```

Comment:
There isn't much benefit to annotating the aggregate disk size in the queryset. We should add a method or property on VirtualMachine to return the aggregate size of assigned VirtualDisks.

Reply:
left this one in as having the property interferes with the same name for the list view queryset on the table, so would need to have the property and the calculated value in the queryset different which seems overly complicated.  I think using aggregation here is cleanest.